HW_7/task34.py

```diff
@@ -0,0 +1,12 @@
+# Задача 34
+
+poem = input("Введите стихотворение: ")
+poem = poem.lower()
```

Comment:
Метод `lower` можно применять сразу.
```suggestion
poem = input("Введите стихотворение: ").lower()
poem = poem
```